README.md

```diff
@@ -296,6 +296,8 @@ render(
 );
 ```
 
+# Code Splitting
+[Code splitting](https://webpack.js.org/guides/code-splitting/) can be used with certain elements. As of now, the elements which support code splitting are `ContentPreview` and `ContentSidebar`. In order to use a code splitted element, you will need to build the element with webpack by importing the element from the `lib` folder in our npm package.
```

Comment:
>[Code splitting](https://webpack.js.org/guides/code-splitting/) is currently support for the `ContentPreview` and `ContentSidebar` UI Elements. In order to use one of these Elements, ...

Reply:
`In order to use an element with code splitting`?